Sources/ff-ios-client-sdk/CfClient.swift

```diff
@@ -627,12 +627,34 @@ public class CfClient {
                     Logger.log("Message: \(decoded)")
                     
 					self.lastEventId = decoded.event
-					self.featureRepository.getEvaluationById(decoded.identifier ?? "", target: self.target.identifier, useCache: false, onCompletion: { (result) in
-						switch result {
-							case .failure(let error): onEvent(EventType.onEventListener(nil), error)
-							case .success(let evaluation): onEvent(EventType.onEventListener(evaluation), nil)
-						}
-					})
+                    
+                    // Handle Target Segmetn Events
```

Comment:
This is one of my typos 
```suggestion
                    // Handle Target Segment Events.  On an Event we need to fetch all evaluations
```